backend/src/validation.rs

```diff
@@ -47,6 +51,10 @@ impl fmt::Display for ValidationResultCode {
             ValidationResultCode::AboveThreshold => write!(f, "Above threshold"),
             ValidationResultCode::EqualInput => write!(f, "Equal input"),
             ValidationResultCode::IncorrectCandidatesList => write!(f, "Incorrect candidates list"),
+            ValidationResultCode::MissingRecounts => write!(f, "Missing recounts"),
+            ValidationResultCode::IncorrectDifference => write!(f, "Incorrect difference"),
+            ValidationResultCode::WrongDifferences => write!(f, "Wrong differences"),
+            ValidationResultCode::NoDifference => write!(f, "No difference"),
```

Comment:
The other codes state what the problem is with the input. For `NoDifference` that would imply no difference was inputted. However, the validation for this code is that at least one difference field was inputted, when it shouldn't have.

Unfortunately, I don't have a suggestion yet for a better name for the code. Will think some more on it.

Reply:
Best I can come up with today is to also make this an `IncorrectDifference`, because a value was filled in where that value should be `0`.

Reply:
Maybe `NoDifferencesExpected`?

Reply:
Done in ed1c100539a85fb761d3dd72d650e5481f6854ea! Was struggling to find an understandable code for this